### PR TITLE
Begin draft of the July newsletter

### PR DESCRIPTION
**Pull request by milancurcic**
Will merge on July 1. Please review, edit, and approve before then.

If you make edits, please also add your name to the Authors list.

### CODE REVIEW

**Comment by milancurcic** [comment]
Assigned @certik for generating the contributors list.

File: _posts/2020-07-01-Fortran-Newsletter-July-2020.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+* [#209](https://github.com/fortran-lang/stdlib/pull/209)
+Implements Simpson's rule for 1-d arrays (`simps` and `simps_weights`) in the
+quadrature module (`stdlib_experimental_quadrature`).
+* [#205](https://github.com/fortran-lang/stdlib/pull/205)
+Tests for and improved standard conformance.
+* [#207](https://github.com/fortran-lang/stdlib/pull/207)
+[#208](https://github.com/fortran-lang/stdlib/pull/208)
+[#210](https://github.com/fortran-lang/stdlib/pull/210)
+[#211](https://github.com/fortran-lang/stdlib/pull/211)
+Miscellaneous fixes and improvements.
```
Comment (jvdp1) [inline _posts/2020-07-01-Fortran-Newsletter-July-2020.md:53]:
```suggestion

Some miscellaneous fixes and improvements:

* [#208](https://github.com/fortran-lang/stdlib/pull/208)
Fixes to support Intel Fortran compilers
* [#210](https://github.com/fortran-lang/stdlib/pull/210)
Fixes to support NAG compiler
* [#207](https://github.com/fortran-lang/stdlib/pull/207)
[#211](https://github.com/fortran-lang/stdlib/pull/211)
Other minor fixes and improvements
```

Comment (jvdp1) [reply _posts/2020-07-01-Fortran-Newsletter-July-2020.md:53]: Just to highlight that other compilers than gfortran are tested.

**Comment by milancurcic** [comment]
#build_preview

File: _posts/2020-07-01-Fortran-Newsletter-July-2020.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Fortran newsletter: July 2020"
+category: newsletter
+author: Milan Curcic, Laurence Kedward
```
Comment (milancurcic) [inline _posts/2020-07-01-Fortran-Newsletter-July-2020.md:5]:
```suggestion
author: Milan Curcic, Laurence Kedward, and Jérémie Vandenplas
```

**Comment by certik** [comment]
I'll update the data files either tonight or tomorrow.

**Comment by milancurcic** [comment]
@certik Sounds good. Whenever you get to update the data, check the newsletter that it looks good, add your edits and name to the list of authors, and merge. IMO it's good to go.

**Comment by certik** [comment]
Didn't get to it today. Do you need this before your presentation, or can I do it some time tomorrow when I have time?

**Comment by LKedward** [comment]
#build_preview

File: _posts/2020-07-01-Fortran-Newsletter-July-2020.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+We thank everybody who contributed to fortran-lang in the past month by
+commenting in any of the four repositories:
+
+* [fortran-lang/stdlib](https://github.com/fortran-lang/stdlib),
+* [fortran-lang/fpm](https://github.com/fortran-lang/fpm),
+* [fortran-lang/fortran-lang.org](https://github.com/fortran-lang/fortran-lang.org),
+* [j3-fortran/fortran_proposals](https://github.com/j3-fortran/fortran_proposals):
+
+TODO: update json datafiles.
```
Comment (jvdp1) [inline _posts/2020-07-01-Fortran-Newsletter-July-2020.md:113]:
```suggestion


```

To be removed ;)

Comment (LKedward) [reply _posts/2020-07-01-Fortran-Newsletter-July-2020.md]: Thanks!

**Comment by LKedward** [comment]
> I'll update the data files either tonight or tomorrow.

I'll move the data files out to a separate repo this month and set up a github action so we don't have to do this manually in future.

**Comment by milancurcic** [comment]
@certik I don't need it for the talk, please do it when it's convenient for you.

**Comment by LKedward** [comment]
@certik, @milancurcic, I pushed updated json data files this morning.

**Comment by certik** [comment]
+1 to merge. Just a note that if we change the contributors section, we need to update all the newsletter with the new code.

File: _posts/2020-07-01-Fortran-Newsletter-July-2020.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+* We had our second Fortran Monthly call on June 19.
+You can read watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/i-gRNGRzugc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+As usual, subscribe to the [mailing list](https://groups.io/g/fortran-lang) and/or
+join the [Discourse](https://fortran-lang.discourse.group) to stay tuned with the future meetings.
+
+* [FortranCon 2020](https://tcevents.chem.uzh.ch/event/12) begins tomorrow (July 2),
```
Comment (LKedward) [inline _posts/2020-07-01-Fortran-Newsletter-July-2020.md:99]:
```suggestion
* [FortranCon 2020](https://tcevents.chem.uzh.ch/event/12) began today (July 2),
```

**Comment by milancurcic** [comment]
Merging, thank you all!

**Comment by LKedward** [comment]
#delete_preview

**Pull request merged**